spyder/plugins/maininterpreter/plugin.py

```diff
@@ -90,6 +91,19 @@ def on_statusbar_available(self):
         if statusbar:

```

Comment:
This if is not necessary because `statusbar` should be available before running this method.